Block/BaseHyvaGrid.php

```diff
@@ -42,29 +37,31 @@ public function getGrid(): HyvaGridInterface
         return $this->grid;
     }
 
-    public function getChildHtml($alias = "", $useCache = false): string
+    public function getNavigationHtml(): string
```

Comment:
Die Methoden `getNavigationHtml`, `getActionsHtml` und getExportsHtml` sollten in das HyvaGridViewModel, nicht in die Block Klasse.
Die grundlegende Idee ist ja so wenig neue Methoden wie Möglich in der Block-Klasse zu haben.